Makefile

```diff
@@ -135,9 +135,14 @@ dist/package.json : src/js/package.json
 npm-link: dist/package.json
 	cd src/test-js && npm ci && npm link ../../dist
 
-dist/pyodide.d.ts: src/js/*.ts src/js/pyproxy.gen.ts src/js/error_handling.gen.ts
-	npx dts-bundle-generator src/js/pyodide.ts --export-referenced-types false
-	mv src/js/pyodide.d.ts dist
+dist/pyodide.d.ts dist/pyodide/ffi.d.ts: src/js/*.ts src/js/pyproxy.gen.ts src/js/error_handling.gen.ts
```

Comment:
It seems like ffi.d.ts is not bundled when creating npm package: https://app.circleci.com/pipelines/github/hoodmane/pyodide/4659/workflows/3f881873-23be-422c-9100-99711b652e16/jobs/62596/parallel-runs/0/steps/0-104

Reply:
Good catch, thanks!